src/test/kotlin/com/igorwojda/integer/reverse/challenge.kt

```diff
@@ -2,9 +2,10 @@ package com.igorwojda.integer.reverse
 
 import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
+import kotlin.math.absoluteValue
 
 private fun reverseInt(i: Int): Int {
-    TODO("not implemented")
+    return i.absoluteValue.toString().reversed().toInt() * if(i<0) -1 else 1
```

Comment:
Instead of checking for the Int value, we can get the -1 or 1 value by using `Int.sign` extension method. It does the same thing, but it looks better. 

Can you also come up with a method that does not convert the `Int` to `String`?